release/ReleaseNotes

```diff
@@ -3,8 +3,29 @@
 
 
 --------------------------------------------------------------------------------------------------------------
-ganga/python/Ganga
+ganga/python/GPIDev
 --------------------------------------------------------------------------------------------------------------
-* ...
+* Fix to MassStorageFile. The local area is now successfully cleaned up after files have been copied
```

Comment:
Also fixed bug where there was a duplication of files when looping over job.outputfiles due to the use of multiple wildcards.